packages/store/src/transformer/job.ts

```diff
@@ -136,11 +136,15 @@ export class Job {
       model,
       assets: this._assetsManager,
     });
-    const children = await Promise.all(
-      model.children.map(child => {
-        return this._blockToSnapshot(child);
-      })
-    );
+
+    const children =
+      model.role === 'root'
```

Comment:
What does this mean? refactoring?  It will make an empty export.

Reply:
Nice catch. It's a bug. And i is already fixed in thttps://github.com/toeverything/blocksuite/pull/5692